gomock/controller.go

```diff
@@ -105,6 +105,26 @@ func NewController(t TestReporter) *Controller {
 	return ctrl
 }
 
+// NewOverridableController returns a new Controller that allows for overridable call expectations
+func NewOverridableController(t TestReporter) *Controller {
```

Comment:
Rather than exposing a separate constructor, let's use functional option pattern here to configure the behavior of the Controller. 

If you rebase off of main, you'll be able to use `ControllerOption` which was added in #24.

Reply:
thanks! updated.